Answers.md

```diff
@@ -0,0 +1,11 @@
+1. Semantic tags: HTML tags that are more litteral and descriptive to make websites more search and developer friendly.
```

Comment:
I like that you took time to answer this questions. Next time consider answering them like if you were explaining the concept to a 12 year old. This way you can see where the real gaps in your understanding are and review them later. https://fs.blog/2012/04/learn-anything-faster-with-the-feynman-technique/